plugin.video.youtube/resources/lib/youtube/helper/yt_login.py

```diff
@@ -25,17 +25,18 @@ def _do_login(_for_tv=False):
         text += '[CR]%s [B]%s[/B]' % (context.localize(provider.LOCAL_MAP['youtube.sign.enter_code']), user_code)
         dialog = context.get_ui().create_progress_dialog(
             heading=context.localize(provider.LOCAL_MAP['youtube.sign.in']), text=text, background=False)
-
+        
+        i=0
```

Comment:
ok let me talk to h0d3nt3uf3l and make sure he doesn’t plan to use it in the future.

Thanks
Jeff



> On May 17, 2016, at 9:21 AM, Razzeee <notifications@github.com> wrote:
> 
> In plugin.video.youtube/resources/lib/youtube/helper/yt_login.py <https://github.com/xbmc/repo-plugins/pull/480#discussion_r63519834>:
> 
> > @@ -25,17 +25,18 @@ def _do_login(_for_tv=False):
> >          text += '[CR]%s [B]%s[/B]' % (context.localize(provider.LOCAL_MAP['youtube.sign.enter_code']), user_code)
> >          dialog = context.get_ui().create_progress_dialog(
> >              heading=context.localize(provider.LOCAL_MAP['youtube.sign.in']), text=text, background=False)
> > -
> > +        
> > +        i=0
> Just a side note, this seems to be unused.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r63519834>